src/components/avatar-rectangle/styles.ts

```diff
@@ -0,0 +1,77 @@
+import { styled } from '@linaria/react'
+import { ReactComponent as ResidentialPlaceholderIcon } from './icons/residential-placeholder-icon.svg'
+import { ReactComponent as CommercialBottomPlaceholderIcon } from './icons/bottom-commercial-placeholder.svg'
+import { ReactComponent as CommercialSmallBottomPlaceholderIcon } from './icons/bottom-commercial-placeholder-small.svg'
+
+const baseCommercialMediumImagesStyle = `
+  width: 54px;
+  height: var(--size-size-8, 40px);
+  flex-shrink: 0;
+`
+
+const baseCommercialBottomPlaceholderStyle = `
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`
+
+export const ElAvatarRectBottomPlaceholder = styled(CommercialBottomPlaceholderIcon)`
+  ${baseCommercialMediumImagesStyle}
+  ${baseCommercialBottomPlaceholderStyle}
+`
+export const ElAvatarRectBottomPlaceholderSmall = styled(CommercialSmallBottomPlaceholderIcon)`
+  ${baseCommercialMediumImagesStyle}
+  ${baseCommercialBottomPlaceholderStyle}
+`
```

Comment:
The way I handle the commercial version is by using a single bottom placeholder (an SVG that looks like a reversed 'L' character) for both the normal and placeholder versions. This approach allows us to provide only two SVGs (the normal and small bottom placeholders) instead of four (the previous SVGs plus two complete normal and small SVGs), since the top placeholder (a grey box SVG) can be replaced with a `div`.

Reply:
**suggestion:** This approach feels more complicated than its worth to me. It doesn't seem like we gain much by minimising the number of SVGs we're dealing with; instead, we are paying for it with more complexity. We need to write code for humans (our future selves, or future others), not computers. Right now, I have no idea what would need to change if Design decided to update how commercial properties are shown; in contrast, if there was a commercial-specific SVG for each size, it's trivial for me to understand what needs to change.

Reply:
Yes, I hadn't thought about that scenario before, but now I understand it. Thank you